cmd/collector/app/span_handler_builder.go

```diff
@@ -58,6 +55,15 @@ func (b *SpanHandlerBuilder) BuildHandlers() (
 		Options.DynQueueSizeMemory(b.CollectorOpts.DynQueueSizeMemory),
 	)
 
+}
+
+// BuildHandlers builds span handlers (Zipkin, Jaeger)
+func (b *SpanHandlerBuilder) BuildHandlers(spanProcessor processor.SpanProcessor) (
+	handler.ZipkinSpansHandler,
+	handler.JaegerBatchesHandler,
+	*handler.GRPCHandler,
```

Comment:
we may want to consider returning a single `*struct SpanHandlers` with 3 handlers